test/app-tap/cfg.test.lua

```diff
@@ -39,5 +39,4 @@ box.cfg{read_only=true}
 while f_ro:status() ~= "dead" do fiber.sleep(0.01) end
 test:is(f_ro:status(), "dead", "entered read-only mode")
 
-test:check()
-os.exit(0)
+os.exit(test:check() and 0 or 1)
```

Comment:
I'd rather introduce something common (something like `test:exit()`) which would be both:
- calling `test:check()` if it was not called yet;
- and exit properly via `os.exit`

Instead of those multiple changes in each relevant file. Though if we consider them pretty much ideomatic then it's ok for me.